### PR TITLE
[DO NOT MERGE] Bug 1170235 compat tables more

### PR DESCRIPTION
**Pull request by darkwing**
Leaving this here, temporarily...

### CODE REVIEW

File: media/stylus/components/compat-tables/bc-legend.styl [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
+legend
+====================================================================== */
+
+.bc-legend {
+    -moz-column-count: 2;
+    -webkit-column-count: 2;
+    column-count: 2;
```
Comment (mdn-services) [inline media/stylus/components/compat-tables/bc-legend.styl]: CSS3 Multiple column layout not supported by: Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3)

File: media/stylus/components/compat-tables/bc-table.styl [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    /*
+    Flexbox for those lucky few who can handle it
+    - we only needs this to make equal height columns in the tablet display view
+    - ie is not among those that can handle this flexbox code,
+      but it doesn't supoort @support so we need to include that in the MQ
+    - all browsers which support @support also use unprefixed syntax
+    ----------------------------------------------------------------------  */
+
+    .bc-table,
+    .bc-table thead,
```
Comment (mdn-services) [inline media/stylus/components/compat-tables/bc-table.styl]: Flexible Box Layout Module not supported by: IE (10), IE Mobile (10)

**Comment by darkwing** [comment]
Closing this as we sort it all out.

File: media/stylus/components/compat-tables/bc-legend.styl [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/*
+legend
+====================================================================== */
+
+.bc-legend {
+    vendorize(column-count, 2);
+    page-break-inside: avoid;
```
Comment (mdn-services) [inline media/stylus/components/compat-tables/bc-legend.styl]: CSS3 Multiple column layout not supported by: Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3)

Comment (mdn-services) [inline media/stylus/components/compat-tables/bc-legend.styl]: CSS3 Multiple column layout not supported by: Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3)

**Pull request closed without merging**